includes/create-theme/theme-styles.php

```diff
@@ -25,7 +25,7 @@ public static function update_style_css( $style_css, $theme ) {
 		$author        = stripslashes( $theme['author'] );
 		$author_uri    = $theme['author_uri'];
 		$wp_version    = CBT_Theme_Utils::get_current_wordpress_version();
-		$wp_min        = $current_theme->get( 'RequiresWP' );
+		$wp_min        = isset( $theme['requires_wp'] ) && $theme['requires_wp'] ? $theme['requires_wp'] : '6.0';
```

Comment:
If  $theme['requires_wp'] is not defined we could defaut to the wordpress version currently in use instead to an arbitrary one:


```suggestion
		$wp_min        = isset( $theme['requires_wp'] ) && $theme['requires_wp'] ? $theme['requires_wp'] : CBT_Theme_Utils::get_current_wordpress_version();
```

Reply:
Done here https://github.com/WordPress/create-block-theme/pull/715/commits/f983c4e2dc630fe81a7b36fb25eee6a506b8533d